### PR TITLE
FE-826/FE-861 Add xGlympseAgent header string

### PR DESCRIPTION
**Pull request by lara-glympse**
FE-826 / FE-861

@Glympse/web PTAL

### CODE REVIEW

File: app/src/GlympseAdapter.js [hunk 1]
```diff
@@ -37,6 +38,9 @@ define(function(require, exports, module)
 
 		var glympserLoader = null;
 
+		ajax.appName = cfg.app.id;
+		ajax.version = cfg.app.version;
+
```
Comment (adubovsky) [inline app/src/GlympseAdapter.js]: I would set whole `requestHeaders` or `commonHeaders` object here as it's done in the viewer.

**Comment by adubovsky** [comment]
Hey Lara, I see this work is related with https://jira.glympse.com/browse/FE-861 task, but needs some adjustments based on the description. The main things are:
- do not use `id`/`version` from the `app` section of config but add params same as in the viewer
- add `comp`/`comp_ver` to the header

**Comment by lara-glympse** [comment]
? I wasn't aware of FE-861, thanks for pointing that out.

This was based on FE-826, which is confusing for only saying [viewer] in title, but says viewer and adapter in description. I'll update to follow FE-861 instead

**Comment by lara-glympse** [comment]
PTAnotherL @adubovsky

File: app/src/GlympseAdapter.js [hunk 2]
```diff
@@ -36,6 +39,22 @@ define(function(require, exports, module)
 		var that = this;
 
 		var glympserLoader = null;
+		var version = VersionInfo.version;
+
+		var localDefaults = {
```
Comment (j5kay) [inline app/src/GlympseAdapter.js]: Do we need this object? It doesn't minify as well and appears to only be a local var?

Comment (lara-glympse) [inline app/src/GlympseAdapter.js]: We don't need it for this particular case. I was following the viewer example and allowing room for potential growth if we started adding more defaults to the adapter. 

I can take it out if that isn't a likely need

File: app/src/GlympseAdapter.js [hunk 3]
```diff
@@ -25,9 +26,11 @@ define(function(require, exports, module)
 
 	function GlympseAdapter(controller, cfg)
 	{
+		var glympseAdapter = 'GlympseAdapter';
```
Comment (adubovsky) [inline app/src/GlympseAdapter.js]: I would prefer `adapterLabel` or smth similar as var name.

**Comment by j5kay** [comment]
As @adubovsky points out, the `README.md` should be updated with info on the new vars available for hosting apps. Additionally, a few other items:
- Version should be updated (2.5.0 -- in `Gruntfile.js`, `bower.json`, and `pacakage.json`)
- New build generated (just need to run `grunt`)
- Update the `CHANGELOG.md` with a blurb on the version/changes
- Tag the commit with the version change as "2.5.0"
- Generate a new release on github.com/Glympse/glympse-adapter for the 2.5.0 build (once all the updates have been committed/merged to master)

**Comment by j5kay** [comment]
Also, updating all relevant apps is documented in FE-873

**Comment by layton-glympse** [comment]
lgtm

**Pull request merged**